pkg/rest/apisurface.go

```diff
@@ -171,15 +171,15 @@ func unpackDeprovisionRequest(r *http.Request) (*osb.DeprovisionRequest, error)
 func (s *APISurface) LastOperationHandler(w http.ResponseWriter, r *http.Request) {
 	version := getBrokerAPIVersionFromRequest(r)
 	if err := s.BusinessLogic.ValidateBrokerAPIVersion(version); err != nil {
-		writeError(w, err)
+		writeError(w, err, http.StatusPreconditionFailed)
 		return
 	}
 
 	request, err := unpackLastOperationRequest(r)
```

Comment:
Oops figured out I forgot to call `unmarshalRequestBody` in the `unpackLastOperationRequest`, can create a PR for that, or if you want to change it here. Up to you?

Just figured out that there would be no errors ever returned here, which made me think why :woman_facepalming:.

Reply:
Let's do that one in a follow-up

Reply:
Last Operation doesn't actually have a request body to deserialize - all the information is carried in query parameters, so we're good from that angle.